go/service/phone_numbers.go

```diff
@@ -0,0 +1,117 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/keybase/client/go/gregor"
+	"github.com/keybase/client/go/libkb"
+	gregor1 "github.com/keybase/client/go/protocol/gregor1"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-framed-msgpack-rpc/rpc"
+
+	"golang.org/x/net/context"
+)
+
+type PhoneNumbersHandler struct {
```

Comment:
if we have a new file and new type for handling phone numbers, can we also move 
```
func (h *UserHandler) AddPhoneNumber(ctx context.Context, arg keybase1.AddPhoneNumberArg)
func (h *UserHandler) VerifyPhoneNumber(ctx context.Context, arg keybase1.VerifyPhoneNumberArg)
func (h *UserHandler) GetPhoneNumbers(ctx context.Context, sessionID int)
```
here, from `service/user.go`

Reply:
excellent idea.